graphite_api/app.py

```diff
@@ -305,8 +305,10 @@ def render():
             request_options['jsonp'] = RequestParams['jsonp']
     if 'maxDataPoints' in RequestParams:
         try:
-            request_options['maxDataPoints'] = int(
+            mdp = int(
                 float(RequestParams['maxDataPoints']))
+            request_options['maxDataPoints'] = mdp
+            g.mdp = mdp
```

Comment:
can we call this `g.maxDataPoints`.   there is no benefit to making this an acronym. When i see `g.mdp` being used in the finder code it is not going to be obvious what it is.